packages/config/tests/mutate/tests.js

```diff
@@ -30,7 +37,7 @@ const initFixtureDir = async function (fixtureName) {
   const fixtureRedirectsPath = `${fixtureDir}/_redirects`
   const redirectsPath = `${fixtureDir}/test_redirects`
   await Promise.all([copyIfExists(fixtureConfigPath, configPath), copyIfExists(fixtureRedirectsPath, redirectsPath)])
-  return { configPath, redirectsPath }
+  return { configPath, redirectsPath, buildDir: fixtureDir }
```

Comment:
These tests started failing randomly after I added the temp files cleanup.

I believe `buildDir` is missing, so the temp directory was always `undefined/.netlify/deploy` which was shared across tests

Reply:
Good catch.